src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Headers.cs

```diff
@@ -557,6 +557,7 @@ public async Task SendAsync_CustomResponseEncodingSelector_CanReceiveNonAsciiHea
         }
 
         [Fact]
+        [SkipOnPlatform(TestPlatforms.Browser, "Content-Length can't be present with Transfer-Encoding")]
```

Comment:
```suggestion
        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotNodeJS))]
```

The test works on browser, we can disable it just for NodeJS.